.github/workflows/testing.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           python-version: '3.10'
       - name: Install Dependencies
         run: |
-          pip install numpy Cython scipy
+          pip install setuptools>=77.0 numpy Cython scipy
```

Comment:
Currently the build step in this workflow won't respect the bound in `pyproject.toml`, so it has to be specified by hand. This could be streamlined in a future PR.